Core/Core/AudioVideo/AudioPlayerViewController.swift

```diff
@@ -177,6 +177,11 @@ public class AudioPlayerViewController: UIViewController {
             play()
         }
     }
+
+    private func formatTime(_ value: TimeInterval) -> String? {
+        let correctValue = value.isFinite ? value : 0
```

Comment:
According to Apple's documentation of property `isFinite`:

> **Discussion**
All values other than NaN and infinity are considered finite, whether normal or subnormal. For NaN, both isFinite and isInfinite are false.

There will be no need to check for `isNaN`.